nrfconnect-toolchain/Dockerfile

```diff
@@ -30,8 +30,10 @@
 
 FROM ubuntu:21.04
 
-ARG TOOLCHAIN_URL="https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2019q4/gcc-arm-none-eabi-9-2019-q4-major-x86_64-linux.tar.bz2"
```

Comment:
Is this change necessary? Isn't adding `-L` to `curl` sufficient?

Reply:
yes -L is sufficient. I have removed -L first, then updated the URL to the redirect one and afterwards added -L because we check the hash now.
Should I use the old "more official" one again?

Reply:
Yes, please. I think the redirect URL is more likely to change, so let's use the old one.